src/scss/components/_ratings.scss

```diff
@@ -0,0 +1,157 @@
+/**
+* Rating element
+* ===================================
+*
+* Selectable star rating
+*
+*   <div class="c-rating-wrapper">
+*       <input id="rating6small" class="c-ratings-input " type="radio" value="6" name="small">
+*       <label for="rating6small" title="6 stars out of 6">6/6</label>
+*       <input id="rating5small" class="c-ratings-input " type="radio" value="5" name="small">
+*       <label for="rating5small" title="5 stars out of 6">5/6</label>
+*       <input id="rating4small" class="c-ratings-input " type="radio" value="4" name="small">
+*       <label for="rating4small" title="4 stars out of 6">4/6</label>
+*       <input id="rating3small" class="c-ratings-input " type="radio" value="3" name="small">
+*       <label for="rating3small" title="3 stars out of 6">3/6</label>
+*       <input id="rating2small" class="c-ratings-input " type="radio" value="2" name="small">
+*       <label for="rating2small" title="2 stars out of 6">2/6</label>
+*       <input id="rating1small" class="c-ratings-input " type="radio" value="1" name="small">
+*       <label for="rating1small" title="1 stars out of 6">1/6</label>
+*   </div>
+*
+* -------------------------------
+*
+* Non-Selectable star rating
+*
+*   <div class="c-ratings ">
+*      <span class="c-ratings--fill c-ratings--fill-18"></span>
+*   </div>
+*
+* ================================= */
+
+$star-filled: "data:image/svg+xml;charset=utf-8,%3Csvg xmlns='http://www.w3.org/2000/svg' width='13' height='13' viewBox='0 0 13 13'%3E%3Cpath fill='%23D50525' d='M6.5 10.8l-4 2.2.8-4.7L0 5l4.5-.7 2-4.3 2 4.3L13 5 9.8 8.3l.7 4.7z'/%3E%3C/svg%3E";
```

Comment:
Icons should either use the SVG map or the `inline()` function and the icons themselves should live in [`f-icons`](https://github.com/justeat/f-icons)